torch/_inductor/ir.py

```diff
@@ -258,6 +258,19 @@ def get_stride_order(seq: Sequence[Union[int, torch.SymInt, Expr]]) -> Sequence[
     return out
 
 
+def get_new_stride_with_stride_order(
+    stride: Sequence[Union[int, torch.SymInt, Expr]],
+    stride_order: Sequence[int],
+) -> Sequence[Union[int, torch.SymInt, Expr]]:
+    """
+    Convert strides order to new stride
+    """
+    new_stride = [stride[0] for _ in range(len(stride))]
```

Comment:
There is a linter check error. so, set all value to stride[0]. maybe wrong here.